components/apim/main.tf

```diff
@@ -6,7 +6,7 @@ module "ctags" {
 }
 
 module "api-mgmt" {
-  source                         = "git::https://github.com/hmcts/cnp-module-api-mgmt-private.git?ref=add-route-table"
+  source                         = "git::https://github.com/hmcts/cnp-module-api-mgmt-private.git?ref=main"
```

Comment:
This has already been resolved, can you pull master into this branch?
https://github.com/hmcts/azure-platform-terraform/blob/master/components/apim/main.tf#L9

Reply:
Update added to validate changes made to apim module.  Will update back to main once module PR is merged.